servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -335,12 +337,19 @@ protected void startServingWebServer() {
   protected void startServingRPCServer() {
     // set up multiplexed thrift processors
     TMultiplexedProcessor processor = new TMultiplexedProcessor();
-    processor.registerProcessor(mBlockMaster.getServiceName(), mBlockMaster.getProcessor());
-    processor.registerProcessor(mFileSystemMaster.getServiceName(),
-        mFileSystemMaster.getProcessor());
-    processor.registerProcessor(mRawTableMaster.getServiceName(), mRawTableMaster.getProcessor());
+    for (Map.Entry<String, TProcessor> service : mBlockMaster.getServices().entrySet()) {
```

Comment:
since we're doing this 4 times it might be worth a `registerProcessors` helper method to register the entries in a `Map<String, TProcessor>`.